docs/features/ImportExport.md

```diff
@@ -81,3 +81,22 @@ By specifying the desired format in the header, you signal the gateway to provid
 Additionally, if you wish to retrieve records and map them to a different-looking object with a Mapping, you can provide the id of that Mapping in the request headers as: x-mapping: {id}.
 
 Ensure you replace {id} with the actual ID of the Mapping object you want to use.
+
+### Export a single object
+The gateway is also able to export single objects to downloadable formats, being PDF, docx and html. However, to be able to do this we need to do a bit of configuration:
+
+First, we should make a template that contains a html/twig template to render the object in the desired format. This can be done in the Admin-UI under the tab ‘Templates’ (and click on add template). Also, a simple example can be found below:
```

Comment:
we once agreed to use Gateway UI for consistency
instead of Admin-UI

Reply:
Thanks, wasn't sure which one we agreed on.